docs/database/sql-server.md

```diff
@@ -152,6 +154,37 @@ and SHOULD be provided **at span creation time** (if provided at all):
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Context propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SET CONTEXT_INFO
+
+Instrumentations MAY propagate context using [SET CONTEXT_INFO](https://learn.microsoft.com/en-us/sql/t-sql/statements/set-context-info-transact-sql?view=sql-server-ver16) by setting a string representation of [`traceparent`](https://www.w3.org/TR/trace-context/#traceparent-header) before executing a query. This is an opt-in behavior that SHOULD be explicitly enabled by the user.
```

Comment:
```suggestion
Instrumentations MAY propagate context using [SET CONTEXT_INFO](https://learn.microsoft.com/sql/t-sql/statements/set-context-info-transact-sql) by injecting fixed-length part of span context (trace-id, span-id, trace-flags, protocol version) before executing a query. For example, when using W3C Trace Context, only a string representation of [`traceparent`](https://www.w3.org/TR/trace-context/#traceparent-header) SHOULD be injected. This is an opt-in behavior that SHOULD be explicitly enabled by the user. 

Variable context parts (`tracestate`, `baggage`) SHOULD NOT be injected since `CONTEXT_INFO` value length is limited to 128 bytes.
```

Reply:
Using a different format, such as B3 without the key, would be tricky for the SQL Server receiver to parse since the format is not clear to the parser.

Imagine a SQL Server database that can serve queries from different SQL clients with different propagation settings. And, only one version of sqlserver receiver fetches the SQL Server that needs to handle both B3 and traceparent.

This would create compatibility issues.

Reply:
Suggestion applied per https://github.com/open-telemetry/semantic-conventions/pull/2363#discussion_r2173106389.